config/initializers/seeds.rb

```diff
@@ -1,5 +1,10 @@
 return unless (ForemanInternal.table_exists? rescue(false)) && !Foreman.in_rake? && !Rails.env.test?
 
+if ActiveRecord::Base.connection.migration_context.needs_migration?
+  Rails.logger.warn("Migrations pending, skipping seeding. Please run `foreman-rake db:migrate` manually.")
+  return
```

Comment:
Should we halt startup here given migrations are needed?

Reply:
halting startup in initialization causes passenger to fail sending users to the http logs, which is why we are adding the second part of this commit. 
The user will see they have pending migrations right after start up thanks to https://github.com/theforeman/foreman/blob/develop/app/controllers/concerns/foreman/controller/migration_checker.rb